app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,52 @@
+    <string name="pref_description_allowOrientation">Permet de changer l\'orientation entre portrait et paysage</string>
+    <string name="pref_title_headerbtm">Placer les onglets en bas</string>
+    <string name="pref_description_headerbtm">Placer les onglets des catégories en bas de l\'écran plutôt qu\'en haut</string>
+    <string name="pref_title_scrollableTabs">Onglets défilants</string>
+    <string name="pref_description_scrollableTabs">Si vous avez beaucoup de catégories (expérimental)</string>
+    <string name="pref_header_appearance">Apparence</string>
+    <string name="pref_title_wallpaperParallax">Défilement du fond d\'écran</string>
+    <string name="pref_description_wallpaperParallax">Déplacer le fond d\'écran lors du changement de catégorie</string>
+    <string name="pref_title_headerDivider">Afficher le séparateur d\'onglet</string>
+    <string name="pref_description_headerDivider">Afficher une ligne blanche entre les onglets et les applications</string>
+    <string name="pref_title_bgGradientColorFromWallpaper">Couleur des ombres d\'arrière plan</string>
+    <string name="pref_description_bgGradientColorFromWallpaper">Colorez l\'ombre de l\'arrière-plan en fonction de la couleur du fond d\'écran (nécessite Android Oreo ou une version plus récente).</string>
+    <string name="pref_title_bgGradientOpacity">Opacité de l\'ombre de l\'arrière plan</string>
+    <string name="pref_header_categoryorder">Ordre des catégories</string>
+    <string name="pref_title_orderreset">Ordre alphabétique</string>
+    <string name="error_oreo_user_handle">L\'utilisateur est verrouillé ou n\'est pas lancé</string>
+    <string name="error_oreo_activity_not_found">L\'activité n\'existe pas</string>
+    <string name="error_oreo_shortcuts_security">Impossible de charger le raccourcis car Fast Draw n\'est pas votre lanceur par défaut</string>
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est définit comme lanceur par défaut.</string>
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est le lanceur par défaut.</string>
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est définit comme lanceur par défaut.</string>
+    <string name="error_expand_notifications_panel">Impossible d\'ouvrir le panneau des notifications. Votre version d\'Android ne supporte peut-être pas cela avec les lanceurs tiers</string>
+    <string name="error_launch_exception">Impossible lancer l\'élément à cause d\'une erreur inconnue. Veuillez signaler ce problème.</string>
+    <string name="error_expand_notifications_panel">Impossible d\'ouvrir le panneau des notifications. Votre version d\'Android ne supporte peut-être pas cela avec les lanceurs tiers</string>
```

Comment:
There are multiple entries for `error_oreo_shortcut_host_permission` and `error_expand_notifications_panel`.

Reply:
Je pense avoir eu un problème avec Android Studio. Je n'ai pas fait attention lors du commit. Le problème est corrigé. Merci.